src/ttkbootstrap/__main__.py

```diff
@@ -93,9 +93,9 @@ def change_theme(e):
     check2.pack(side=LEFT, expand=YES, padx=5)
 
     check4 = ttk.Checkbutton(rb_group, text="deselected")
-    check4.pack(side=LEFT, expand=YES, padx=5)    
+    check4.pack(side=LEFT, expand=YES, padx=5)
+    check4.invoke()
     check4.invoke()
```

Comment:
Are there supposed to be two check4.invoke() calls here?

Reply:
Yes I can verify that the base repo has the same logic:
https://github.com/israel-dryer/ttkbootstrap/blob/master/src/ttkbootstrap/__main__.py

Might be because "deselected" means selecting and unselecting.